packages/styles/src/components/list-group.scss

```
@@ -38,125 +24,87 @@
   }
 
   .list-group-item-right-svg {
+    padding-block: tokens.get('list-icon-container-padding-block');
+    padding-inline: tokens.get('list-icon-container-padding-inline');
+
     @include transition(
       transform animation.$transition-time-area-small animation.$transition-easing-default
     );
   }
-
-  &-minimal,
-  &-transparent {
-    &:hover,
-    &:focus {
-      background-color: transparent;
-    }
-  }
-
-  &-inverted {
-    background-color: transparent;
-
-    &:hover,
-    &:focus {
-      background-color: list-group.$list-group-hover-bg-inverted;
-    }
-  }
 }
 
 .list-group-item {
-  display: flex;
-  align-content: center;
-  align-items: center;
-  justify-content: space-between;
-  padding-right: map.get(spacing.$post-sizes, 'small-regular');
-  padding-left: map.get(spacing.$post-sizes, 'small-regular');
-  background-color: transparent;
-  font-weight: normal;
-  text-decoration: none;
-  border-collapse: collapse;
-  border-right: 0;
-  border-left: 0;
-
-  @include media-breakpoint-up(rg) {
-    padding-left: map.get(spacing.$post-sizes, 'large');
-
-    &-minimal {
-      padding-left: 0;
-    }
-
-    &-transparent {
-      padding-right: map.get(spacing.$post-sizes, 'mini');
-      padding-left: map.get(spacing.$post-sizes, 'regular');
+  border: unset;
+  padding: 0;
+  background: transparent;
+
+  > a,
+  .form-check {
```

Would be nice, if we could extract these styles and provide them without the need to be nested in the `.list-group > li`. This would allow us using them also on other places (for example in the <post-list> web-component).

```suggestion
  .list-group-link,
  .list-group-document,
  .list-group-switch {
```

---

Also adopt the styles defined below for these elements.